src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -401,6 +401,14 @@ impl LocalStateStore for LocalHummockStorage {
             next_epoch,
             prev_epoch
         );
+        self.event_sender
```

Comment:
It doesn't seem to be working, do we need to send the event in the next pr?

Reply:
yes